.cargo/config.toml

```diff
@@ -30,4 +33,8 @@ rustflags = [
 	"-Aclippy::module_name_repetitions",            # It seems we prefer it this way; we'd need to discuss that
 	"-Aclippy::must_use_candidate",                 # Overzealous, we'd have to `[must_use]` a lot of things
 	"-Aclippy::redundant_closure_for_method_calls", # Not always clearer, let's not pepper `allow`s whenever needed
+	# Nursery
```

Comment:
I think we did a great push in Q4 with linting, so I'm not suggesting we spend more time on this. However, I think there are a few things remaining in #155 so I would like to keep it open for now. If you agree, we can unassign it and push it out of this sprint, or split the issue:

- panics: while we cannot prevent it 100%, especially in depenedencies, we can still prevent the majority of it (panic/unwrap/expect/etc...) in our code, which I think is more likely to have panics. I would like to try turning it on just in `outputs/cargo/crate/.cargo/config.toml` and `outputs/npm/crate/.cargo/config.toml`. The rest of the codebase is internal, and panicing there is fine.
- Not sure if we reviewed [allowed-by-default](https://doc.rust-lang.org/rustc/lints/listing/allowed-by-default.html) already for anything useful?